build/scripts/meta.yaml.schema

```diff
@@ -118,7 +118,19 @@
               },
               "memoryLimit": {
                 "type": "string",
-                "pattern": "[1-9][0-9]*([Gg]i|(Mm)i|(Kk)i|b)?"
+                "pattern": "^[1-9][0-9]*([Gg]i?|[Mm]i?|[Kk]i?|b)?$"
+              },
+              "memoryRequest": {
+                 "type": "string",
+                 "pattern": "^[1-9][0-9]*([Gg]i?|[Mm]i?|[Kk]i?|b)?$"
+              },
+              "cpuLimit": {
+                 "type": "string",
+                 "pattern": "(^[0-9]+(.[0-9]+)?$|^[1-9][0-9]*(m)$)"
```

Comment:
1. dot means any character, so 2x2 is a valid cpuLimit )
2. Brackets at the beginning and the end are redundant
3. Do you really need a separate group for `m` char?
```suggestion
                 "pattern": "^[0-9]+(\\.[0-9]+)?$|^[1-9][0-9]*m$"
```
Also this expression does not cover requirement 
> precision finer than 1m is not allowed. For this reason, the form 100m might be preferred
Up to you to cover it or not

Reply:
Seems the same changes are required for `initContainers`

Reply:
About `initContainers`.. do we need some code changes in Che server ho handle that ?

Reply:
I don't know